com.stremio.App.json

```diff
@@ -0,0 +1,175 @@
+{
+    "app-id": "com.stremio.App",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "version": "24.08",
+            "directory": "lib/ffmpeg",
+            "add-ld-path": "."
+        },
+        "org.freedesktop.Platform.VAAPI.Intel": {
+            "version": "24.08",
+            "directory": "lib/intel-vaapi-driver",
+            "add-ld-path": ".",
+            "download-if": "have-intel-gpu",
+            "enable-if": "have-intel-gpu"
+        }
+    },
+    "command": "stremio",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home/.stremio-server:create",
+        "--filesystem=xdg-data/stremio:create",
+        "--filesystem=xdg-run/stremio:create",
```

Comment:
why do you need access to these two?

Reply:
`xdg-data/stremio` used for cache
`xdg-run/stremio` used for unix socket

Reply:
what do you mean by honor `XDG_*` ?

Reply:
Can you explain what is the issue exactly?